docs/mast/mast.rst

```diff
@@ -324,35 +286,77 @@ Product filtering can also be applied directly to a table of products without pr
                 >>> print(len(products))
                 4
 
+
+Downloading Data Products
+-------------------------
+
+Products can be downloaded by using `~astroquery.mast.ObservationsClass.download_products`,
+with a `~astropy.table.Table` of data products, or a list (or single) obsid as the argument.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Observations
```

Comment:
this looks overindented

Reply:
Just checked the rest of `mast.rst` and the indentation is consistent with the rest of the document. I worry pulling it back will mess up the formatting.

Reply:
OK, then this is an issue for a separate quick cleanup PR.